ethereum/ethevents/events.go

```diff
@@ -290,13 +290,36 @@ func (ev *EthereumEvents) SubscribeEthereumEventLogs(ctx context.Context, fromBl
 		go ev.runEventProcessor(ctx)
 	}
 
+	// check eth connection
+	connCh := make(chan bool, 1)
```

Comment:
This channel should have a more descriptive name, such as "connFailure"

Reply:
changed :)